cpg_workflows/stages/large_cohort.py

```diff
@@ -1,16 +1,19 @@
 from cpg_utils import Path
 from cpg_utils.config import get_config
+from cpg_utils.hail_batch import image_path, query_command
+
 from cpg_workflows.targets import Cohort
 from cpg_workflows.utils import slugify
 from cpg_workflows.workflow import (
-    stage,
     CohortStage,
     StageInput,
     StageOutput,
     get_workflow,
+    stage,
 )
-from .genotype import Genotype
+
 from .. import get_batch
```

Comment:
🤔 technically this is a possible import, but it's not an input from source. This should be an import from `cpg_utils.hail_batch`